src/design-documentation/project-structure.md

```diff
@@ -52,16 +52,41 @@
   - **unit** — Unit tests using rustc’s built-in test harness.
   - **wpt** — W3C web-platform-tests and csswg-tests along with tools to run them and expected failures.
 
-# Major dependencies
+# Repositories the Servo project maintains
 
-* <https://github.com/servo/mozjs>: bindings to SpiderMonkey
-* <https://github.com/hyperium/hyper>: an HTTP implementation
-* <https://github.com/servo/html5ever>: an HTML5 parser
-* <https://github.com/servo/ipc-channel>: an IPC implementation
-* <https://github.com/image-rs/image>: image decoders
-* <https://github.com/rust-windowing/winit>: cross-platform windowing and input
-* <https://github.com/linebender/vello>: a pure Rust 2D graphics library
-* <https://github.com/servo/rust-cssparser>: a CSS parser
-* <https://github.com/housleyjk/ws-rs>: a WebSocket protocol implementation
-* <https://github.com/servo/rust-url>: an implementation of the URL specification
-* <https://github.com/servo/webrender>: a GPU renderer
+The Servo project maintains a number of repositories that are either independent from Servo or forked from some upstream project with ongoing updates.
+
+## Repositories that are used throughout the rust ecosystem
+
+- [ipc-channel](https://github.com/servo/ipc-channel): Crate for the rust ecosystem of making IPC work.
+- [string-cache](https://github.com/servo/string-cache): String interning library.
+- [surfman](https://github.com/servo/surfman): Low-Level cross platform Rust library for managing graphic surfaces.
+
+## Repositories that are forked from upstream projects
+
+- [euclid](https://github.com/servo/euclid): Geometric types.
+- [mozjs](https://github.com/servo/mozjs): Spidermonkey, servos JavaScriptEngine based on Spidermonkey with a small patchset.
+- [rust-url](https://github.com/servo/rust-url): Url type.
+- [rust-content-security-policy](https://github.com/servo/rust-content-security-policy): Parse and validate Content Security Policy.
+- [stylo](https://github.com/servo/stylo): Servos CSS engine based on Firefox CSS engine.
+- [unicode-bidi](https://github.com/servo/unicode-bidi): Unicode Bidirectional Algorithm implementation.
```

Comment:
We don't use this and I have no idea why we forked it, so let's remove this (and maybe the fork??).